ui_framework/components/pager/pager_button_group.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { KuiButton, KuiButtonIcon, KuiButtonGroup } from '../button';
+
+export function KuiPagerButtonGroup({ className, onPrevious, onNext, hasNext, hasPrevious, ...rest }) {
+  return (
+    <KuiButtonGroup isUnited className={className} { ...rest }>
+      <KuiButton
+        aria-label="Show previous page"
+        data-test-subj="pagerPreviousButton"
+        type="basic"
+        onClick={onPrevious}
+        disabled={!hasPrevious}
+        icon={<KuiButtonIcon type="previous" />}
```

Comment:
Can we add `aria-hidden="true"` here? This will prevent screen readers from trying to read the Font Awesome icon, since it's technically text, just using an icon font. If/when we switch to SVG icons this won't matter.